src/test/java/bio/terra/flight/ProfileDeleteFlightTest.java

```diff
@@ -0,0 +1,40 @@
+package bio.terra.flight;
+
+import bio.terra.common.category.Unit;
+import bio.terra.service.profile.flight.delete.ProfileDeleteFlight;
+import bio.terra.stairway.FlightMap;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.context.ApplicationContext;
+
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@RunWith(MockitoJUnitRunner.StrictStubs.class)
+@Category(Unit.class)
+public class ProfileDeleteFlightTest {
+
+    @Mock
+    private ApplicationContext context;
+
+    @Test
+    public void testConstructFlight() {
+        var flight = new ProfileDeleteFlight(new FlightMap(), context);
+
+        var packageName = "bio.terra.service.profile.flight.delete";
+
+        assertThat(flight.context().getStepClassNames(), is(List.of(
+                packageName + ".DeleteProfileMarkUnusedProjects",
+                packageName + ".DeleteProfileDeleteUnusedProjects",
+                packageName + ".DeleteProfileProjectMetadata",
+                packageName + ".DeleteProfileMetadataStep",
+                packageName + ".DeleteProfileAuthzIamStep"
+        )));
```

Comment:
I really dig this approach!  Given the conditional step addition stuff I did for azure in the last PR, this gives a great way to test that!  Small style thing: I think we typically append the closing parens on the last non-paren line (e.g. you'd have:
```
assertThat(flight.context().getStepClassNames(), is(List.of(
                packageName + ".DeleteProfileMarkUnusedProjects",
                packageName + ".DeleteProfileDeleteUnusedProjects",
                packageName + ".DeleteProfileProjectMetadata",
                packageName + ".DeleteProfileMetadataStep",
                packageName + ".DeleteProfileAuthzIamStep")));
```
I believe this comes from the google codestyle standard